views/features.njk

```diff
@@ -0,0 +1,184 @@
+            </div>
+          </div>
+          <div class="usa-width-one-third">
+            <div class="feature-group vertical">
+              <div class="feature-image small">
+                <img src="/images/icons/icon-layout-color.svg" />
+              </div>
+              <div class="feature-text">
+                <h4>Accessible templates</h4>
+                <p><a href="https://federalist-docs.18f.gov/pages/using-federalist/templates/">Our 508-compliant website templates</a> help you kickstart your design process.</p>
+              </div>
+            </div>
+          </div>
+          <div class="usa-width-one-third">
+            <div class="feature-group vertical">
+              <div class="feature-image small">
+                <img src="/images/icons/icon-badge-color.svg" />
+              </div>
+              <div class="feature-text">
+                <h4>Customization</h4>
+                <p>If you have an in-house design team, they can use a static site generator like Jekyll, Hugo, or their own tools to automatically deploy a custom site built just for you.</p>
+              </div>
+            </div>
+          </div>
+         </div>
+        <div class="usa-grid-full">
+          <div class="usa-width-one-third">
+            <div class="feature-group vertical">
+              <div class="feature-image small">
+                <img src="/images/icons/icon-clockback-color.svg" />
+              </div>
+              <div class="feature-text">
+                <h4>Simple version control</h4>
+                <p>Use GitHub, a web-based hosting service with built-in version control, for real-time publishing and easy collaboration</p>
+              </div>
+            </div>
+          </div>
+          <div class="usa-width-one-third">
+            <div class="feature-group vertical">
+              <div class="feature-image small">
+                <img src="/images/icons/icon-two-squares-color.svg" />
+              </div>
+              <div class="feature-text">
+                <h4>Reusable and public</h4>
+                <p>The code used for Federalist is open source and on the public domain for anyone to reuse. Share code with other agencies and vendors also using Federalist.</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+  <div class="usa-grid">
+    <hr class="hr-light">
+    <section class="info-block">
+      <h2>It’s easy to get started.</h2>
+      <p>We’ve streamlined our onboarding process to get you working quickly:</p>
+      <ul>
+        <li>No hidden charges.</li>
+        <li>All the storage and bandwidth you need.</li>
+        <li>Unlimited additional sites for your team or office.</li>
+        <li>Two weeks to execute an average contract.</li>
+      </ul>
+    </section>
+  </div>
+  <div class="well">
+    <center>
+      <strong>Try a free test site to see if Federalist is right for you.</strong>
```

Comment:
This needs to be larger than regular body copy, and should keep the bold font weight. In mockups, I used the `h5`,  but that might not be semantically appropriate. What else might we be able to use?